src/main/java/uk/gov/companieshouse/company/metrics/config/MongoConfig.java

```diff
@@ -42,11 +44,13 @@ public ObjectMapper customMapper() {
         // Exclude properties with null values from being serialised
         objectMapper.configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
         objectMapper.configure(SerializationFeature.FAIL_ON_EMPTY_BEANS, false);
+        objectMapper.configure(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS, false);
         objectMapper.setSerializationInclusion(JsonInclude.Include.NON_NULL);
         SimpleModule module = new SimpleModule();
         module.addSerializer(LocalDate.class, new LocalDateSerializer());
         module.addDeserializer(LocalDate.class, new LocalDateDeSerializer());
         objectMapper.registerModule(module);
+        objectMapper.registerModule(new JavaTimeModule());
```

Comment:
This nullifies above LocalDate custom serialisers, should remove it? if needed we should more custom serialisers for LocalDateTime or OffsetTime etc

Reply:
removed the java time module and customized the LocalDateTime Serializer/Deserializer and made sure the code uses the custom deserializer.